shared/signup/phone-number/phone-input.tsx

```diff
@@ -218,8 +218,10 @@ type Props = {
   autoFocus?: boolean
   defaultCountry?: string
   onChangeNumber: (phoneNumber: string) => void
-  onChangeValidity: (valid: boolean) => void
+  onChangeValidity: (valid: boolean, phoneNumber: string) => void
```

Comment:
Might make sense to include `valid` in `onChangeNumber` and get rid of this callback. maybe add a TODO?

Reply:
Refactored to remove the callback.